src/components/Table.jsx

```diff
@@ -0,0 +1,376 @@
+    };
+
+    const handleAssign=(e)=>{
+        setSearchByAssignee(e.target.value)
+    }
+
+    const handleStatus = (e) => {
+        setIsStatusSelected(true)
+        setSearchByStatus(e.target.value)
+    }
+
+    const titleRowFilterTemplate = () => {
+        return (<span className="p-input-icon-left w-[100%] min-w-[80px]"><i className="pi pi-search" /><InputText onChange={handleTitleChange} /></span>);
+    };
+
+    const brandRowFilterTemplate = () => {
+        return (
+                <MultiSelect
+                value={selectedBrand}
+                options={brands}
+                itemTemplate={representativesItemTemplate}
+                onChange={(e) => handleBrands(e.value)}
+                optionLabel="brand"
+                placeholder="Brand"
+                maxSelectedLabels={1}
+                style={{width: "100px"}}
+            />
+        );
+    };
+
+    const updatedByFilterTemplate=()=>{
+        return (
+                <span className="p-input-icon-left w-[100%] min-w-[80px]">
+                    <i className="pi pi-search" />
+                    <InputText onChange={handleUpdatedByChange}/>
+                </span>
+        );
+    }
+
+    const styleRowFilterTemplate = () => {
+        return (
+                <span className="p-input-icon-left w-[100%] min-w-[80px]">
+                    <i className="pi pi-search" />
+                    <InputText  onChange={handleStyleChanges} />
+                </span>
+        );
+    };
+
+    const dateFilterTemplate = () => {
+        return (
+                <span className="p-input-icon-left w-[100%] min-w-[80px]">
+                    <Calendar value={searchByUpdatedAt} onChange={handleCalanderChange} placeholder='dd/mm/yyy'/>
+                </span>
+        )
+    };
+
+
+    const handleCalanderChange = (e) => {
+        setsearchByUpdatedAt(e.target.value)
+    }
+
+    const dateBodyTemplate = (rowData) => {
+        return formatDate(rowData.lastUpdateTs);
+    };
+
+    const formatDate = (value) => {
+        var date = new Date(value);
+        const newDate = date.toDateString().split(' ')
+        const finalDate = `${newDate[0]} ${newDate[2]} ${newDate[1]}`
+        return finalDate
+    };
+
+   const handleFilterIcon=()=>{
+      return(
+        <span>
+            <button onClick={() => setShowFilters(!showFilters)}>
+                 {!showFilters ? <div className="bg-black text-white text-sm rounded-full border h-8 w-8 flex justify-center items-center"> 
+                     <img alt={`${FilterIcon} svg`} src={FilterIcon}/>
+                 </div> :
+                 <div className="bg-white text-black text-sm rounded-full border h-8 w-8 flex justify-center items-center"> 
+                    <img alt={`${FilterIconBlack} svg`} src={FilterIconBlack}/>
+                 </div>}
+            </button>                
+        </span> 
+    )
+}
+{/* <span>
+<button onClick={() => setShowFilters(!showFilters)}>
```

Comment:
remove commented lines if not needed.

Reply:
I removed it in the previous PR only I am not able to see this in my code